public/apps/account/test/account-nav-button.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('Account navigation button', () => {
 
   const config = {
     multitenancy: {
-      enabled: 'true',
+      enabled: true,
```

Comment:
Is this change necessary or just for consistency with other settings?

Reply:
This is a straight forward change within the file that other changes are being made in - I think this is good to include

Reply:
@cliu123 This was actually a *bug* in the existing test. While the string 'true' evaluates to truth-y in js, it did not work when setting the reverse to 'false' since the string 'false' is also truth-y in js. I had to make a separate config to test for multitenancy: false so I went back up to this config and switched from a string to a bool as well.

Reply:
@cwperks Thanks for clariying! That's a great finding! line 35 and 36 look like the same bug. Could you also fix them?
```
        enable_private: 'true',
        enable_global: 'true',
```

Reply:
@cliu123 This is updated now.

Reply:
Cool! Thanks!